index.html

```diff
@@ -3,11 +3,24 @@
   <head>
     <meta charset="utf-8">
     <title>jQuery</title>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
```

Comment:
scripts move to the end of document

also remember that your js should load after jquery file